### DISCUSSION TITLE
120Hz and blurry text for 14"/16" M1?

### DISCUSSION THREAD

**Discussion by AwlsomeAlex** [utmapp/UTM#3435] [UTM Help (macOS)]
For Linux VMs (specifically Ubuntu 21.10), is there any way to configure UTM so it can take advantage of the ProMotion displays? Also is there a way to configure it to use native resolution/scaling so the text isn't as blurry?

**Comment by korayJB** [comment]
ı don't think it is possible but if you do it, please let me know.

**Reply by korayJB** [reply]
by the way ı can not connect my usb wifi dongle to my virtualbox. can you help me with that. it says failed to attach usb dongle to the virtualbox. ım using 5370 chipset.

**Comment by adespoton** [comment]
In response to the original question, it's not a UTM issue but an Ubuntu ARM issue.
The following might help:
https://www.reddit.com/r/linux4noobs/comments/rmm4s8/ubuntu_always_boots_in_60hz_by_default_on_a_120hz/
As for blurry text, part of that is the Ubuntu text renderer, and part of it is not being in Retina mode.  Double your screen display and then enable Retina Mode in the UTM Displays settings.  You'll obviously have to then adjust icon and font display size appropriately.
